### PR TITLE
Actions: add stale to auto close old stalled PR

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

**Comment by TingPing** [comment]
Well that was easy. Maybe 6 months is reasonable?

**Comment by barthalion** [comment]
A year is good enough IMHO, we can always change it later.

**Comment by bilelmoussaoui** [comment]
A year sounds good to me too.  Any feedback regarding the current message?

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token:  ${{ secrets.FLATHUBBOT_TOKEN }}
+        stale-pr-message: "This PR didn't receive any update in a year and will be automatically closed in 14 days."
```
Comment (TingPing) [inline .github/workflows/stale.yml]:
```suggestion
        stale-pr-message: "This PR hasn't receive any update in a year and will be automatically closed in 14 days."
```

Comment (barthalion) [reply .github/workflows/stale.yml]: It should also say something along "Feel free to re-open it if you plan to continue working on that pull request or think it deserves attention from Flathub admins".

Comment (bilelmoussaoui) [reply .github/workflows/stale.yml]: Done

**Pull request merged**